### PR TITLE
Fix link to OWNERS_ALIASES

### LINKED ISSUE #31279: `OWNERS_ALIASES` links go to `OWNERS`, which is wrong
**This is a Bug Report**

**Problem:**
In [here](https://kubernetes.io/docs/contribute/participate/roles-and-responsibilities/#becoming-a-reviewer) and [here](https://kubernetes.io/docs/contribute/participate/roles-and-responsibilities/#becoming-an-approver),
`OWNERS_ALIASES` links go to https://github.com/kubernetes/website/blob/main/OWNERS, which is wrong.

They should go to https://github.com/kubernetes/website/blob/main/OWNERS_ALIASES .

**Proposed Solution:**
Find `[OWNERS_ALIASES](https://github.com/kubernetes/website/blob/main/OWNERS)`,
and replace with `[OWNERS_ALIASES](https://github.com/kubernetes/website/blob/main/OWNERS_ALIASES)` .

**Page to Update:**
https://kubernetes.io/docs/contribute/participate/roles-and-responsibilities/

/language en
/kind cleanup
/good-first-issue

### PR DESCRIPTION
**Pull request by championshuttler**
Fixes #31279

### CODE REVIEW

**Review by mehabhalodiya — COMMENTED** [review]
LGTM!

File: content/en/docs/contribute/participate/roles-and-responsibilities.md [hunk 1]
```diff
@@ -147,7 +147,7 @@ separately for reviewer status in SIG Docs.
 To apply:
 
 1. Open a pull request that adds your GitHub user name to a section of the
-   [OWNERS_ALIASES](https://github.com/kubernetes/website/blob/main/OWNERS) file
+   [OWNERS_ALIASES](https://github.com/kubernetes/website/blob/main/OWNERS_ALIASES) file
```
Comment (wiggitywhitney) [inline content/en/docs/contribute/participate/roles-and-responsibilities.md:150]: @bradtopol I investigated these links (also line 222) and the changes look good to me

Comment (bradtopol) [reply content/en/docs/contribute/participate/roles-and-responsibilities.md:150]: Great catch!

Comment (bradtopol) [inline content/en/docs/contribute/participate/roles-and-responsibilities.md:150]: /assign
/lgtm
/approve

**Pull request merged**